prow/plugins/wip/wip-label.go

```diff
@@ -55,10 +55,17 @@ func init() {
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// Only the Description field is specified because this plugin is not triggered with commands and is not configurable.
-	return &pluginhelp.PluginHelp{
-			Description: "The wip (Work In Progress) plugin applies the '" + label + "' label to pull requests whose title starts with 'WIP' and removes it from pull requests when they remove the title prefix. The '" + label + "' label is typically used to block a pull request from merging while it is still in progress.",
-		},
-		nil
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: "The wip (Work In Progress) plugin applies the '" + label + "' label to pull requests whose title starts with 'WIP' and removes it from pull requests when they remove the title prefix. The '" + label + "' label is typically used to block a pull request from merging while it is still in progress.",
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
```

Comment:
This plugin is controlled by changing the title of the PR, it doesn't actually have a command so this plugin is fine as is.

Reply:
done.